src/main/native/cpp/view/Logger.cpp

```diff
@@ -74,6 +66,56 @@ void Logger::Display() {
   ImGui::TextColored(m_ntConnected ? kColorConnected : kColorDisconnected,
                      m_ntConnected ? "NT Connected" : "NT Disconnected");
 
+  // Create a Section for project configuration
+  ImGui::Separator();
+  ImGui::Spacing();
+  ImGui::Text("Project Parameters");
+
+  // Add a dropdown for mechanism type.
+  ImGui::SetNextItemWidth(ImGui::GetFontSize() * 10);
+
+  if (ImGui::Combo("Mechanism", &m_selectedType, kTypes,
+                   IM_ARRAYSIZE(kTypes))) {
+    m_settings.mechanism = analysis::FromName(kTypes[m_selectedType]);
+  }
+
+  // Add Dropdown for Units
+  ImGui::SetNextItemWidth(ImGui::GetFontSize() * 10);
+  if (ImGui::Combo("Unit Type", &m_selectedUnit, kUnits,
+                   IM_ARRAYSIZE(kUnits))) {
+    m_settings.units = kUnits[m_selectedUnit];
+  }
+
+  sysid::CreateTooltip(
+      "This is the type of units that your robot will be using."
+      "For example, if your flywheel will be using rotations, then use the "
```

Comment:
I think we should say something like "if you want your flywheel gains in terms of radians, use the radians unit."